CharacterCreation/01_03_Races_Tundari

```diff
@@ -0,0 +1,171 @@
+==============================
+Tundari
+==============================
+
+===== Society =====
+
+The Tundari are a race of obligate carnivores from the planet Tundaar. Physically, 
+they span a wide size range, from the Chi’mau, who can range from 3’6” to 5’ and 
+85-130 lbs to the Sang’mau, at 6’ to 7’ and up to 400 lbs, and every size in-between. 
+Like humans, body types persist in many forms. 
+A sleek coat of fur covers these people, ranging widely in color and pattern. It 
+tends to grow short and fine over the majority of the body, however, the face, head, 
+neck, and joints have been known to grow hair that is longer and coarser, having a 
+mane or scruff-like quality about it. 
+As many cultures mark themselves with tattoos, the Tundari also have their own method
+of skin-marking. Some outliers will dye their fur, but that is generally frowned upon 
+by much of Tundari society. The traditional method of marking is called Aranga, 
+though many ranchers will would recognize the method as very similar to the freeze-
+branding they mark livestock with. Aranga, however, is far more intricate that simple 
+freeze-branding.
+As a carnivorous species, they have large, forward, wide-set eyes, clawed limbs and 
+large, sharp teeth. Their ears sprout from closer to the top of their heads than the 
+sides, and can be triangular or round. 
+They are considered a hardy species, but have not adapted well to fight off poisons 
+or foreign contaminants. They are diurnal, and have low-light eyesight, but can 
+also see well in daylight. 
+
+Culturally and economically, the clans on the Tunadri homeworld have an instated 
+breeding season once every four years. Cubs born outside of this season are not 
+harmed, but their parents are firmly reprimanded for the first, and drastic measures 
+may be taken if a single family has multiple children outside of the typical breeding 
+times. Rather than the byproduct of a controlling government, these mandates came 
+into effect nearly four-hundred years prior. As infant mortality rates decreased, the
+population increased far more rapidly than their ecosystem could handle. They have 
+kept their population stable for the last three centuries, and no one seems inclined 
+to change their ways. Space exploration and colonization have begun to change that, 
+but many colonists still ascribe to the old mandates.
+
+Types:                            Biological
+                                  Mammalian
+                                  Warm-blooded 
+                                  Carnivorous 
+
+Weakness to Poisons/Contagions:   [Insert Poison rules here]
```

Comment:
I would almost prefer to leave this blank and add an issue to add it later rather then put dev notes in the documents. That said, the dev branch can tolerate dev notes. It's just that it's really easy for them to slip by the RC edit and find their way further up the development tree. Not a blocker.

Reply:
Can I create an issue referencing this particular line so that mechanics can fill it in?

Reply:
ya! If you click on the line number on the left side of the page when you look at the code, it will highlight the line and change the URL in your browser to a link to that particular line that you can paste into the issue.